fastchat/serve/inference.py

```diff
@@ -164,6 +164,9 @@ def generate_stream(
                 skip_special_tokens=True,
                 spaces_between_special_tokens=False,
             )
+            # TODO: For the issue of incomplete sentences interrupting output, apply a patch and others can also modify it to a more elegant way
+            if not is_sentence_complete(output):
+                stopped = False
```

Comment:
I think it is a useful feature. However, it is hard-coded and can influence the model output dramatically. For some applications, we may not want this.
Can we make it an optional argument? It should be turned off by default to reflect the actual model performance.

Reply:
sure